### DISCUSSION TITLE
How to use webwidget_triggered on chatwoot

### DISCUSSION THREAD

**Discussion by CelestineAkpanoko** [chatwoot/chatwoot#4880] [Q&A]
**How to make the chatbot initiate the conversation. So, specifically, I want the chatbot to send a message after the user clicks on the live widget. From the documentation, I understand Chatwoot webhook has a supported event (webwidget_triggered) that accomplishes this task.**

**Comment by pranavrajs** [comment]
Does Campaign work in your use case? 

https://www.chatwoot.com/docs/product/features/campaigns/ongoing

**Reply by CelestineAkpanoko** [reply]
The campaign works but is not exactly what I am looking for. It displays a popup message. However, after a user sends a message, how can I clear or refresh the chat?

**Reply by CelestineAkpanoko** [reply]
It does. Thanks!
